Tests/Runtime/Core/InstalledJavaVersionCheck/InstalledJavaVersionProviderTests.cs

```diff
@@ -35,6 +35,35 @@ public void CheckInstalledJavaVersion_WhenExpectedJavaVersionIsInstalled_IsSucce
             Assert.IsTrue(response.IsInstalled);
         }
 
+        [Test]
+        public void CheckInstalledJavaVersion_WhenExpectedJavaVersionIsMultiline_IsSuccessFul()
```

Comment:
Add a test case to check what happens if `java -version` return empty output

Reply:
Done.